### PR TITLE
movie: Deep Blue Sea (1999)

### PR DESCRIPTION
**Pull request by paul-sheridan**
Genetically engineered sharks swim amok. Rating 2/5.

### CODE REVIEW

File: notes/stories/film/film-scifi-1990s.st.txt [hunk 1]
```diff
@@ -11669,6 +11669,47 @@ factions joining forces against a common enemy [The people of Naboo joined force
...
+:: Genre
+sci-fi
+action
+
+:: Choice Themes
+human vs. beast [The people trapped in the ocean facility were being terrorized by multiple genetically engineered sharks.]
+
+:: Major Themes
+cure for an as yet uncurable disease [Dr. Susan McCallister was genetically engineering mako sharks to have bigger brains as part of her research to help cure Alzheimer's disease.]
+the dangers of unfettered scientific advancement [Dr. Susan McCallister accidentally created a pack of intelligent sharks that swam amok while seeking a cure for Alzheimer's disease.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-1990s.st.txt:11702]: Were the sharks searching for a cure for Alzheimer's?

File: notes/stories/film/film-scifi-1990s.st.txt [hunk 2]
```diff
@@ -11669,6 +11669,47 @@ factions joining forces against a common enemy [The people of Naboo joined force
...
+species uplifting [The genetically engineered mako sharks made a complex plan to escape to the open ocean from the facility that was their prison. In particular, the sharks manipulated people into flooding the ocean facility, so they (the sharks) could escape to through some weak mesh fences at the surface, bypassing the lower titanium walls.]
```
Comment (odinlake) [inline notes/stories/film/film-scifi-1990s.st.txt:11703]: We should consider how this relates to "intelligent animal". Same thing except one is supposedly naturally smart and the other for some help?

**Review by odinlake — APPROVED** [review]
Comments inline

File: notes/stories/film/film-scifi-1990s.st.txt (hunk 1 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1990s.st.txt]: I wish. That would have put an interesting spin on things. But alas not. Comment disambiguated accordingly.

File: notes/stories/film/film-scifi-1990s.st.txt (hunk 2 above)
Comment (paul-sheridan) [reply notes/stories/film/film-scifi-1990s.st.txt:11703]: That's just the conclusion I arrived at after considering which, if any, of those two themes to apply here. Anything to update about the definition in your view?

**Pull request merged**